db/seeds.rb

```diff
@@ -706,3 +706,47 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   )
 end
 
+# ----------------------------------------------------------------------------
+# Counties
+# ----------------------------------------------------------------------------
+Rake::Task['db:load_us_counties'].invoke
+
+# ----------------------------------------------------------------------------
+# Partner Counties
+# ----------------------------------------------------------------------------
+
+# aim -- every partner except one will have some non-zero number of counties,  and the first one 'verified' will have counties, for convenience sake
+# Noted -- The first pass of this is kludgey as all get out.  I'm *sure* there is a better way
+partner_ids = Partner.pluck(:id)
+partner_ids.pop(1)
+county_ids = County.pluck(:id)
+
+partner_ids.each do |partner_id|
+  partner = Partner.find(partner_id)
+  profile = partner.profile
+  num_counties_for_partner = Faker::Number.within(range: 1..10)
```

Comment:
Is there a reason we need to randomize this?

Reply:
Variety.  Slightly more realistic?    More a want than a need, then.

Reply:
In general I don't like randomization in tests because then you are required to work around the resulting flakiness by e.g. asserting that some value is equal to the random value you just generated. It's a bit harder to reason about IMO.

Reply:
Ah.  I see the seeds as being as  much or more for having data to play with when doing manual tests / demonstrations, as for the automatic tests.